packages/server/test/integration/http_requests_spec.coffee

```diff
@@ -2619,20 +2659,6 @@ describe "Routes", ->
               ## shouldn't be more than 500ms
               expect(reqTime).to.be.lt(500)
 
-              # b = res.body
```

Comment:
So... I should have left a note indicating why this stuff is here - back in the day I wanted the ability to make modifications to the JS rewriting code and then test out its affect on performance. 

Back then we didn't have a way to capture the performance timings easily - but now we do. Instead of commenting out this code can we write it out to the performance API? Also, it may be worth converting this into another p95 kind of test - because small changes to the regex can have a dramatic impact on the network performance. We actually caused a severe regression that the tests didn't pick up, which is when I wrote this code.

I think there is some test somewhere but its not super sensitive - it basically just checks to ensure that the tests finish within a given time range threshold, which would catch really bad regressions, but it's still not sensitive enough to tell us small changes to the performance. By capturing all the timings via the Performance API we would know when those small changes occurred.

Reply:
Does it need to happen in this PR? I can open an issue/add this to the proxy refactor branch (https://github.com/cypress-io/cypress/pull/5136) since it also changes the rewriting stuff somewhat

Reply:
yes thats fine